ocaml/typing/persistent_env.ml

```diff
@@ -20,15 +20,28 @@ open Misc
 open Cmi_format
 
 module CU = Compilation_unit
-module Consistbl = Consistbl.Make (CU.Name) (CU)
+module Impl = struct
+  (* The implementation compilation unit for some import, if known *)
+  type t =
+    | Unknown_argument (* The import is a parameter module *)
+    | Known of CU.t
```

Comment:
Honestly, I'd rather change the ones in `Cmi_format.kind`, especially once `-as-argument-for` comes around and adds another field to the `Normal` constructor. They're correct by construction, which feels maybe sensible for a file format but unwieldy for keeping in an environment.

Reply:
I don't follow. This type is currently the same as `Cmi_format.kind` but with worse names for the constructors. This should use the better names.

Reply:
Wound up ditching this whole module in favor of `Import_info.Intf.Nonalias.Sort`, since its whole purpose is to be put into a `Consistbl` anyway and that's what that's for.